Formula/okta-aws-cli.rb

```diff
@@ -0,0 +1,22 @@
+class OktaAwsCli < Formula
+  desc "Okta authentication in support of aws cli operation"
+  homepage "https://github.com/okta/okta-aws-cli"
+  url "https://github.com/okta/okta-aws-cli/archive/refs/tags/v0.2.1.tar.gz"
+  sha256 "06374f0eb3e371a8ef1a5e8fb2bcd0e5bfcac607aa2083f6c1101b54713a47bf"
+  license "Apache-2.0"
+
+  depends_on "go"
```

Comment:
Should be just build dep?
```suggestion
  depends_on "go" => :build
```